examples/Benchmark/generic/scenario_storage.h

```diff
@@ -54,8 +54,7 @@ CURRENT_STRUCT(UInt32KeyValuePair) {
 CURRENT_STRUCT(StringKeyValuePair) {
   CURRENT_FIELD(key, std::string);
   CURRENT_FIELD(value, uint32_t);
-  CURRENT_CONSTRUCTOR(StringKeyValuePair)(const std::string& key = "", uint32_t value = 0)
-      : key(key), value(value) {}
+  CURRENT_CONSTRUCTOR(StringKeyValuePair)(const std::string& key = "", uint32_t value = 0) : key(key), value(value) {}
```

Comment:
Thanks for `make indent`! Perhaps change this to "pass by copy, initialize by move" too?